apis/placement/v1beta1/work_types.go

```diff
@@ -63,6 +68,23 @@ type Manifest struct {
 	runtime.RawExtension `json:",inline"`
 }
 
+type WorkloadOverride struct {
+	// Identifier represents an identity of a resource which has defined the overrides.
+	// +required
+	Identifier WorkResourceIdentifier `json:"identifier"`
+
+	// Overrides defines a list of override applied to the resource before applying to the spoke cluster.
+	// +required
+	Overrides []Override `json:"overrides"`
```

Comment:
I can't seem to find where "Override" is defined

Reply:
It's the full structure defined in the override_types.go.

I was thinking of populating the object metadata, overridePolicies, applyMode and priority fields for the troubleshooting purpose.
It may be too much and could exceed the 1mb limit of the object. 

Now I'm leaning toward creating a new struct to include the object name and its observedGeneration.